### ISSUE TITLE
`make docker` refuses to build because node version is too old

### ISSUE THREAD

**Issue by alejandro-perez** [TheIdentitySelector/thiss-js#251]
```
 make docker
make BASE_URL='$${BASE_URL}' COMPONENT_URL='$${BASE_URL}cta/' MDQ_URL='$${MDQ_URL}' PERSISTENCE_URL='$${BASE_URL}ps/' SEARCH_URL='$${SEARCH_URL}' STORAGE_DOMAIN='$${STORAGE_DOMAIN}' LOGLEVEL='$${LOGLEVEL}' DEFAULT_CONTEXT='$${DEFAULT_CONTEXT}' WHITELIST='$${WHITELIST}' build_in_docker
make[1]: se entra en el directorio '/home/alex/OneDrive/thiss-js'
docker build -t thiss-builder:1.6.4 -f Dockerfile.build .
[+] Building 79.9s (9/10)                                                                                                                                                                
 => [internal] load .dockerignore                                                                                                                                                   0.9s
 => => transferring context: 2B                                                                                                                                                     0.0s
 => [internal] load build definition from Dockerfile.build                                                                                                                          0.7s
 => => transferring dockerfile: 184B                                                                                                                                                0.0s
 => [internal] load metadata for docker.io/library/node:12                                                                                                                          2.3s
 => [1/6] FROM docker.io/library/node:12@sha256:01627afeb110b3054ba4a1405541ca095c8bfca1cb6f2be9479c767a2711879e                                                                   43.6s
 => => resolve docker.io/library/node:12@sha256:01627afeb110b3054ba4a1405541ca095c8bfca1cb6f2be9479c767a2711879e                                                                    0.5s
 => => sha256:01627afeb110b3054ba4a1405541ca095c8bfca1cb6f2be9479c767a2711879e 776B / 776B                                                                                          0.0s
 => => sha256:3a69ea1270dbf4ef20477361be4b7a43400e559c6abdfaf69d73f7c755f434f5 2.21kB / 2.21kB                                                                                      0.0s
 => => sha256:6c8de432fc7f7d8c58899f61982d1662ec6b73fb3ef92f862ba170dcc5b64fa9 7.68kB / 7.68kB                                                                                      0.0s
 => => sha256:f44e4bdb3a6c1325cc4d40e585ed7a759127c0c87b0388ec0236b1698827d70d 4.34MB / 4.34MB                                                                                      1.2s
 => => sha256:9bed1e86f01ee95c76d2c8b4385a47ae336e6d293afade9368469d99daa9369f 11.30MB / 11.30MB                                                                                    9.3s
 => => sha256:f5196cdf25181bc7e4411865a2e002932b7b6b0ffce787c04c1bdeaf1f204f20 45.43MB / 45.43MB                                                                                    9.1s
 => => sha256:2f75d131f4060950dd6cc1f580e2fa5504ece8d692113a9cdb0a866637b397d7 49.77MB / 49.77MB                                                                                   11.8s
 => => extracting sha256:f5196cdf25181bc7e4411865a2e002932b7b6b0ffce787c04c1bdeaf1f204f20                                                                                           0.6s
 => => sha256:07dff4ad21ebdb3ce3e329699663b2f81af70152453025f6624584a39a8e22b6 214.48MB / 214.48MB                                                                                 35.2s
 => => sha256:e0ac4f13b766d321acc3b650d3d23b82828995711f6f247ff591722c00d04cec 4.19kB / 4.19kB                                                                                      9.7s
 => => sha256:df2c3b2eb7cc63351bb32f26457bbe0402af8082548f26975f0c329bc7841881 23.70MB / 23.70MB                                                                                   15.0s
 => => extracting sha256:9bed1e86f01ee95c76d2c8b4385a47ae336e6d293afade9368469d99daa9369f                                                                                           0.2s
 => => extracting sha256:f44e4bdb3a6c1325cc4d40e585ed7a759127c0c87b0388ec0236b1698827d70d                                                                                           0.1s
 => => extracting sha256:2f75d131f4060950dd6cc1f580e2fa5504ece8d692113a9cdb0a866637b397d7                                                                                           0.7s
 => => sha256:efe636eac583776a8a114d50fef15bc65b648f3d2bb53326cf1f21cc5ef2b3ae 2.34MB / 2.34MB                                                                                     12.6s
 => => sha256:fe17849545bb51455d3f7c8773ded2dbb1d6668a85bd00564573a4b88afd36f6 464B / 464B                                                                                         12.9s
 => => extracting sha256:07dff4ad21ebdb3ce3e329699663b2f81af70152453025f6624584a39a8e22b6                                                                                           2.6s
 => => extracting sha256:e0ac4f13b766d321acc3b650d3d23b82828995711f6f247ff591722c00d04cec                                                                                           0.0s
 => => extracting sha256:df2c3b2eb7cc63351bb32f26457bbe0402af8082548f26975f0c329bc7841881                                                                                           0.5s
 => => extracting sha256:efe636eac583776a8a114d50fef15bc65b648f3d2bb53326cf1f21cc5ef2b3ae                                                                                           0.1s
 => => extracting sha256:fe17849545bb51455d3f7c8773ded2dbb1d6668a85bd00564573a4b88afd36f6                                                                                           0.0s
 => [internal] load build context                                                                                                                                                   0.7s
 => => transferring context: 604.81kB                                                                                                                                               0.0s
 => [2/6] WORKDIR /usr/src/app                                                                                                                                                      0.9s
 => [3/6] COPY package*.json webpack* ./                                                                                                                                            0.8s
 => [4/6] RUN npm install -g npm webpack webpack-cli                                                                                                                               29.6s
 => ERROR [5/6] RUN npm install                                                                                                                                                     1.4s 
------                                                                                                                                                                                   
 > [5/6] RUN npm install:                                                                                                                                                                
#0 1.174 ERROR: npm v9.6.0 is known not to run on Node.js v12.22.12. You'll need to                                                                                                      
#0 1.174 upgrade to a newer Node.js version in order to use this version of npm. This                                                                                                    
#0 1.174 version of npm supports the following node versions: `^14.17.0 || ^16.13.0 ||                                                                                                   
#0 1.174 >=18.0.0`. You can find the latest version at https://nodejs.org/.                                                                                                              
#0 1.174 
#0 1.174 ERROR:
#0 1.175 /usr/local/lib/node_modules/npm/lib/utils/exit-handler.js:21
#0 1.175   const hasLoadedNpm = npm?.config.loaded
#0 1.175                            ^
#0 1.175 
#0 1.175 SyntaxError: Unexpected token '.'
#0 1.175     at wrapSafe (internal/modules/cjs/loader.js:915:16)
#0 1.175     at Module._compile (internal/modules/cjs/loader.js:963:27)
#0 1.175     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
#0 1.175     at Module.load (internal/modules/cjs/loader.js:863:32)
#0 1.175     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
#0 1.175     at Module.require (internal/modules/cjs/loader.js:887:19)
#0 1.175     at require (internal/modules/cjs/helpers.js:74:18)
#0 1.175     at module.exports (/usr/local/lib/node_modules/npm/lib/cli.js:81:23)
#0 1.175     at Object.<anonymous> (/usr/local/lib/node_modules/npm/bin/npm-cli.js:2:25)
#0 1.175     at Module._compile (internal/modules/cjs/loader.js:999:30)
------
Dockerfile.build:5
--------------------
   3 |     COPY package*.json webpack* ./
   4 |     RUN npm install -g npm webpack webpack-cli
   5 | >>> RUN npm install
   6 |     COPY src ./src
   7 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c npm install" did not complete successfully: exit code: 1
make[1]: *** [Makefile:77: thiss_builder] Error 1
make[1]: se sale del directorio '/home/alex/OneDrive/thiss-js'
make: *** [Makefile:50: standalone_in_docker] Error 2

```

**Comment by leifj** [comment]
Yes this is known. We are working on a major revision of node and webpack. Should be done in a few days.

**Comment by alejandro-perez** [comment]
Thank. I'll keep monitoring

**Comment by enriquepablo** [comment]
Fixed in version 2.0.0
